front/formdisplay.php

```diff
@@ -70,6 +70,11 @@
                                                 : "'" . $form->fields['entities_id'] . "'";
          $_SESSION['glpilanguage'] = $form->getBestLanguage();
       }
```

Comment:
Check visibility restriction before displaying a form.
I believe this check was missing, a user could still reach a forbidden form if he knew the url.